tests/Adapter/AdapterTestCase.php

```diff
@@ -48,4 +48,16 @@ protected function sourceExpectedAndOffset($manifestPath)
 
         return [ $source, $expected, $offsetStart, $offsetEnd ];
     }
+
+    protected function sourceExpectedAndWordUnderCursor($manifestPath): array
+    {
+        list($source, $expected) = $this->sourceExpected($manifestPath);
+
+        preg_match('/(\w+)<>(\w+)/', $source, $matches);
+        $word = $matches[1] . $matches[2];
+
+        $source = preg_replace('/<>/', '', $source);
+
+        return [ $source, $expected, $word ];
```

Comment:
doesn't seem to be used currently

Reply:
We could keep it so we can still use the same data provider in the test.
If we don't then I feel like we shouldn't use the `generateAccessorX.test` files, because it will enforce a implicit rule to name the tested property `method` and nothing else.

Reply:
Another option would be to extract the "word" from the start / end offsets, if you check the `ExtractOffset::fromSource()` call in `sourceExpected` it returns start and end offsets when applied to:

```
class Foobar
{
    private $<>foobar<>;
}
```

So that might be better? Another option might be to provide parameters as key/value pairs in the test file header.

You're right, we lose some of the value if we don't use the cursor specifiers (`<>`) here, but otherwise I'd be happy to remove them, and just have the pre and post transformation states and have the method name hardcoded.

If we want to validate the method name, then we could create a method name validator and unit test it elsewhere...

Reply:
> Another option would be to extract the "word" from the start / end offsets

I like that, I didn't take enough time to understand the `ExtractOffset::fromSource`
I'm not a big fan of regexp anyway.

> Another option might be to provide parameters as key/value pairs in the test file header.

Not a big fan, but only because Im' not use to process text like that in my tests.
So it feels weird to me to have a new format.
I would almost prefer to have all the code in the test instead.

So I will refactor the `sourceExpectedAndWordUnderTheCursor` to use `sourceExpectedAndOffset`